boards/esp32c3/kitten_grapebit_c3.board.json

```diff
@@ -0,0 +1,61 @@
+{
+    "$schema": "../esp32deviceconfig.schema.json",
+    "devName": "Kittenbot Grape:Bit ESP32-C3 DeviceScript",
+    "productId": "0x3635315b",
+    "$description": "Stemcraft C3",
+    "jacdac": {
+        "$connector": "Jacdac",
+        "pin": 5
+    },
+    "led": {
+        "pin": 8,
+        "activeHigh": true
+    },
+    "i2c": {
+        "$connector": "Qwiic",
+        "pinSCL": 7,
+        "pinSDA": 6
+    },
+    "pins": {
+        "P1": 2
+    },
+    "log": {
+        "pinTX": 0,
+        "baud": 115200
+    },
+    "services": [
```

Comment:
We generally use `$services` with `$` which means the services need to be started explicitly by the user. Otherwise it's difficult to bind the service to the correct button or motor controller. Plain `services` is mostly for things that the user wouldn't really touch, like the power service.

I'll be adding docs

Reply:
actually we seem to already have docs  https://microsoft.github.io/devicescript/devices/add-board#services

Reply:
@mmoskal But the onboard services don't show up in the dashboard with $services define.